03-react-components/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/app.jsx',
```

Comment:
Bei den Pfaden in dieser Datei – sollten wir da nicht mit `path.join` arbeiten, damit es plattformübergreifend funktioniert?

Reply:
Du hast Recht, habe ich übersehen.
Ich habe es einheitlich in der gesamten `webpack.config.js` nachgezogen.